Podfile

```diff
@@ -0,0 +1,25 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '10.0'
```

Comment:
Should this specify the latest platform version instead?

Reply:
Its minimum supported platform. This library is also open sourced so we can support platforms below what we support internally.